### ISSUE TITLE
Too much focus on the file list headings

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud-libraries/nextcloud-vue#4871]
This was discussed during the process of developing the heading toggles but must have fallen off. :)
It would be nice if the file list heading goes to the previous style of:
- Being color-text-maxcontrast
- Not being bold

Otherwise the heading overshadows the actual files and folders:
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/925062/1c8f7a7b-969f-461d-b302-d2184a70a002)

cc @marcoambrosini @szaimen – and also @susnux @ShGKme since you were in the discussions of the headings when we did that in person.

**Comment by susnux** [comment]
This is not part of the vue components but of the files app

**Comment by susnux** [comment]
We could make it easier to have all of our a11y and design decisions implemented for table headers and provide a `NcTableHeader` component that already implements this.

**Comment by marcoambrosini** [comment]
I think they're really nice and help speed up understanding what's what.

Example:
![Screenshot from 2023-11-23 08-59-59](https://github.com/nextcloud-libraries/nextcloud-vue/assets/26852655/7d090ca8-deac-4de9-924f-d941cc52d198)

 If they were maxcontrast and non-bold, they would be indistinguishable from the actual data of the file element. Making things more disorienting to the user.

Especially taking into account that we might have an “owner” and  “share status” columns in the future. I think it makes sense to keep them as they are.

**Comment by jancborchardt** [comment]
Here are all the possibilities, whole viewport:

Bold headings and file names | Current 28 with bold table headings
-|-
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/925062/ad364387-14a3-49ba-aee8-7ad448ac731f)|![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/925062/48b68269-a9b5-43e3-80b8-5b20a1273b75)
Normal font weight but no maxcontrast | Going back to how it was before 
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/925062/ef91ca19-c4f8-45eb-a3c8-fdad270902a1)|![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/925062/9735a2e8-ef69-4044-8e13-98dfd2615694)

- Bold headings and file names: (Just as experiment) This is definitely too much. It’s like seeing one thing that shouldn’t be so present and making the next thing more present, creating overload.
- Current 28 with bold table headings: Well, the table headings are too present for what they are for. It is a pretty simple table, sorting is not going to be changed that often, filename being the most obvious thing – it is just unnecessary to bold it.
- Normal font weight but no maxcontrast: Already better but still the headings have the same importance as the file names which makes little sense
- Going back to how it was before: Well, this is how the proper information architecture should be, and why it was like that before. Headings are not that important.


As a sidenote, mostly as info @nextcloud-libraries/designers: It sometimes happens that there is a code or architecture change, and with it comes an unplanned design change. This happens either because it was an oversight or there wasn’t enough time (or both) – either is fine. When afterwards issues (basically design regressions) are noticed and brought up though, often it already being like that is used as an argument to just roll with it.
I have to say I don’t feel comfortable with this as it feels like design changes are being introduced without thinking about it and then reasoning about them afterwards. We should keep code and design changes separate discussions – especially when it has been discussed and reasoned about before, otherwise it’s very exhausting. Hope that’s understandable. :)

**Comment by marcoambrosini** [comment]
I disagree. I'm looking at the screenshots and still think that it's easier to scan the page when the headings are bold. This is also providing a horizontal "line" that cuts between the above section and files list.

For reference:

Google Drive
![Screenshot from 2023-11-24 09-25-17](https://github.com/nextcloud-libraries/nextcloud-vue/assets/26852655/3518d94c-d4e5-4644-aaa9-4fa122dc06e5)

Onedrive 
![Screenshot from 2023-11-24 09-28-45](https://github.com/nextcloud-libraries/nextcloud-vue/assets/26852655/f214a145-bb13-44a2-baa5-1a1903a3fe39)
